api/data_explorer/controllers/search_controller.py

```diff
@@ -15,28 +18,69 @@ def search_get():
     """
 
     es = Elasticsearch(current_app.config['ELASTICSEARCH_URL'])
-    search = Search(using=es, index=current_app.config['FIELDS_INDEX_NAME'])
-    # TODO(malathir): Change this to case insensitive sorting.
-    search = search.sort('name.keyword')
-    # Default number of results is 10, which isn't enough.
-    search = search[0:10000]
-    response = search.execute()
-    response_fields = response.to_dict()
-
     search_results = []
-    for field in response_fields['hits']['hits']:
-        if "description" in field["_source"]:
-            search_results.append(
-                SearchResult(
-                    facet_name=field["_source"]["name"],
-                    facet_description=field["_source"]["description"],
-                    elasticsearch_field_name=field["_id"],
-                    facet_value=""))
-        else:
-            search_results.append(
-                SearchResult(
-                    facet_name=field["_source"]["name"],
-                    elasticsearch_field_name=field["_id"],
-                    facet_value=""))
+
+    if query:
```

Comment:
nit can you reverse the order this if: `if not query:`

So this follows chronological order.
- First query is empty and we return all fields, to populate search drop-down
- Then user types query and query is set